Main.java

```diff
@@ -1,6 +1,7 @@
+import java.util.Scanner;
 /**
- *
- * @author 
+ * pizza

```

Comment:
There should be a more descriptive title here. Remember, this is so you don't have to go reading your code so you know exactly what this program is for.